### PR TITLE
Update launch screen and welcome content

### LINKED ISSUE #239: Update login button title on welcome screen ("Get Started") when not connected to Firefox Account yet
~Currently, the button always says "Get Started". It should say "Sign in with your Firefox Account" on the welcome screen when the application is locked.~

Changed default back to "Sign in with your Firefox Account" at #244.

Re-framed this issue to be a nice-to-have to introduce logic to change the label to "Get Started" when no Firefox Account is connected (primarily experienced during a first run, and after unlinking, both likely to be rare occurrences).

Welcome ("Get Started"): https://app.zeplin.io/project/5a4e9ff8032d0923200a2635/screen/5a4ea0625387ae000e1a0780

Returning User ("Sign in with your Firefox Account") https://app.zeplin.io/project/5a4e9ff8032d0923200a2635/screen/5ac2933f2cbc9d64992d6273

Related to #223

### LINKED ISSUE #306: Welcome & loading screen content updates
Content updates to the welcome & loading screens based on content review.

## Tasks
**Welcome (first run) / Welcome (returning user) / Loading Screen**
- [x] Update slogan to: “Take your passwords everywhere”
- [x] Update "Lockbox" to: "Firefox Lockbox"

**Welcome (first run)**
- [x] Update copy to: "To use Firefox Lockbox, you’ll need a Firefox Account with saved logins."
- [x] “Learn more [chevron-right]” link opens FAQ to anchored section
Link directs to: https://lockbox.firefox.com/faq.html#how-do-i-use-lockbox

**Welcome (returning user)**
- [x] Update sign in with FxA button to: “Sign In”

## Visual Design
**InVision**
Loading Screen: https://mozilla.invisionapp.com/share/7JEUG47ZU#/267561207_Loading_Screen
Welcome: https://mozilla.invisionapp.com/share/BQEUSO0S2#/267560515_Welcome
Welcome (Returning User - Touch ID): https://mozilla.invisionapp.com/share/7JEUG47ZU#/288717368_Welcome_-_Returning_User_-_TouchID
Welcome (Returning User - Face ID): https://mozilla.invisionapp.com/share/7JEUG47ZU#/267560535_Welcome_-_Returning_User

**Zeplin**
Loading Screen: https://zpl.io/begYxoD
Welcome: https://zpl.io/aBjB70q
Welcome (Returning User - Touch ID) : https://zpl.io/V0ZW5xO
Welcome (Returning User - Face ID): https://zpl.io/aRNL9Ln

### PR DESCRIPTION
**Pull request by devinreams**
Fixes #306 
Fixes #239 

This updates the title, tagline, sign in button, and helpful first run blurb.

screen shot 2018-05-01 at 3 56 27 pm

- [x] It does NOT yet include the added "Learn more >" blurb and link.
  - Still needs to be hooked up to route / actual FAQ link #340 

It also _still_ does not include the tagline on the launch screen as it would not be able to be translated (ie: better to show that which won't change like the title only, then progressively change/add that which can like the tagline).

### CODE REVIEW

**Comment by devinreams** [comment]
OK I've got the "Learn more" button added to the Welcome screen and tappable.

screen shot 2018-05-02 at 2 58 23 pm

It needs somewhere to go and I tried looking at the routes and webview stuff and got confused. 

@sashei perhaps you can help review my code and then we can pair 💻 on the "Learn more" link soon?

**Comment by devinreams** [comment]
>Loading screen still showing just "Lockbox" ❌

Yeah, my original comment was: It also still does not include the tagline on the launch screen as it would not be able to be translated (ie: better to show that which won't change like the title only, then progressively change/add that which can like the tagline).

If you feel strongly right now that we should include it, cool. Just know it likely won't be maintainable/easily do-able as we introduce translations (so any reason to bother now?)

**Comment by nickbrandt** [comment]
@devinreams Sorry I should have been a little more clear. I wasn't referring to the inclusion of the tagline as much as I was referring to updating "Lockbox" to "Firefox Lockbox". I think it's fine that the tagline doesn't exist still, though it feels more important to update the name - for both the legal reasons as well as eliminating the jump between the two names.

**Comment by devinreams** [comment]
>Sorry I should have been a little more clear. I wasn't referring to the inclusion of the tagline as much as I was referring to updating "Lockbox" to "Firefox Lockbox".

Oooh, well then that's very odd. Because it's just a pointer to the exact same image called "lockbox-title" as the one on the Welcome screen. Hmm 🤔 

screen shot 2018-05-03 at 12 42 38 pm

@nickbrandt Maybe it's cached locally for you. Can you please try re-installing the build and see if that cleans it up?

**Comment by nickbrandt** [comment]
@devinreams I deleted and re-installed on my personal device and still shows "Lockbox". However, we tried it on our test device and shows correct "Firefox Lockbox". 😕 Don't know why that's happening, but I assume we are good here?

File: lockbox-ios/Presenter/WelcomePresenter.swift [hunk 1]
```diff
@@ -70,6 +71,10 @@ class WelcomePresenter {
                     .bind(to: view.firstTimeLoginMessageHidden)
                     .disposed(by: self.disposeBag)
 
+            lockedObservable
```
Comment (sashei) [inline lockbox-ios/Presenter/WelcomePresenter.swift]: Rx-y!!! 🙌

**Comment by devinreams** [comment]
This is crashing at the Welcome screen. I think I broke the storyboard. Stay tuned...

**Comment by devinreams** [comment]
>This is crashing at the Welcome screen. I think I broke the storyboard.

Oh wait, _all_ my builds, including against `master` are crashing at Welcome screen. So this isn't this branch. Something likely to do with leftover sync data and biometrics (#314) filed issue: #342 

If I completely reset my simulator this is looking A-OK. ✨ 

iPhone 8
screen shot 2018-05-09 at 3 29 08 pm

iPhone SE
screen shot 2018-05-09 at 3 30 46 pm


Since we're up to date with master, tests are in, code and UI reviews approved: this is ready to rock and roll! 🌊🚢

**Pull request merged**